patches/vst3_paths.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/source/vst/hosting/module_linux.cpp b/source/vst/hosting/module_linux.cpp
```

Comment:
Unfortunately, I had to drop this patch, because nowadays the VST SDK is downloaded during the CMake configure step (that's doubtlessly a very disputable choice...)

Reply:
>  because nowadays the VST SDK is downloaded during the CMake configure step (that's doubtlessly a very disputable choice...)

This won't work. The build must be offline.

Reply:
btw this patch wasn't applied for a reason. VST support has never worked.

Reply:
Looks like we're lucky and there is still an option to use a local VST SDK. I hope I specified all the paths correctly, let's see...